metadata.json

```diff
@@ -35,6 +35,10 @@
     {
       "name": "katello/certs",
       "version_requirement": ">= 6.0.0 < 7.0.0"
+    },
+    {
+      "name": "theforeman/pulpcore",
+      "version_requirement": ">= 0.0.1 < 0.2.0"
```

Comment:
Since we aren't releasing puppet-pulpcore yet, I think this has to get dropped here

Reply:
got it, thanks